modules/nw-egress-ips-object.adoc

```diff
@@ -23,7 +23,7 @@ spec:
 ----
 <1> The name for the `EgressIPs` object.
 
-<2> An array of one or more IP addresses.
+<2> An array of one or more IP addresses. Must be either IPv4 or IPv6. You cannot configure both IPv4 and IPv6 egress IPs in the same `EgressIP` object.
```

Comment:
I think this is not true, we can configure both IPv4 and IPv6 egress IPs in the same `EgressIP` object, they cannot be assigned to same egress node, but they can be assigned to different egress nodes.

@kyrtapz  Can you help confirm?

Reply:
@kyrtapz thanks pat. Is this scenario supported on OCP? 
I didnt see any tests in our upstream code (e2es or units), so i concluded we didnt support it even if the code looks like it handles this scenario. 

@huiran0826 do you have tests testing the mentioned scenario?

Reply:
@kyrtapz @martinkennelly  Thanks for confirming.  I have a case configured this way, but that is for configuration check (https://github.com/openshift/openshift-tests-private/blob/master/test/extended/networking/cloud_egressip_ovn.go#L3212).  Just like Patrk said, no load balance there.   Agree to consider it as not support.